### ISSUE TITLE
Clear Cache Option

### ISSUE THREAD

**Issue by brixly** [AnoopAlias/nDeploy#331]
Would be great to have an option within the cPanel plugin allowing the customer to clear the cache for whichever template they are using (if caching is selected in the template).

For example, if the customer is using the 'Wordpress Full page caching' the 'clear cache' button should remove the redis cache.

If FCGI or micro-caching is selected in the template the same button should clear those caches.

**Comment by AnoopAlias** [comment]
@brixly  - Hi Dennis.. XtendWeb support full page cache/ proxy cache and proxy microcache . The full page cache needs nginx helper and there is option to flush cache in the plugin itself (the cache is redis ) .
 MicroCache need no flushing as this cache the content for like 1 second only. Microcaching is a weak form of caching  . It only makes a content that has a very large number of visit in a few seconds (like a dos attack be cached.

In short the template themselves(or the plugin associated) takes care of the cache purge . The default template doesn't use any caching. the advantage of XtendWeb is smoothening effect and mitigations Xtendweb-nginx applies.
